dashboard/models.py

```diff
@@ -82,22 +81,21 @@ def set_user_defaults(self, course_id, user_id, default_view_type, default_view_
 
 class UserDefaultSelection(models.Model):
     id = models.AutoField(primary_key=True, verbose_name="Table Id")
-    user_id = models.CharField(max_length=255, verbose_name="SIS Id")
-    course_id = models.CharField(max_length=255, verbose_name="Course Id")
-    default_view_type = models.CharField(max_length=255, verbose_name="Default Type")
-    default_view_value = models.CharField(max_length=255, verbose_name="Default Value")
+    course_id = models.CharField(max_length=255, blank=True, null=True, verbose_name="Course Id")
+    user_id = models.CharField(max_length=255, blank=True, null=True, verbose_name="SIS Id")
```

Comment:
I think I mixed this up initially but it's still mixed up
```suggestion
    user_id = models.CharField(max_length=255, blank=True, null=True, verbose_name="User Id")
```

Reply:
fixed